tests/functional_tests/ffront_tests/test_foast_parsing.py

```diff
@@ -259,26 +259,3 @@ def operator_with_refs(inp: Field[..., "float64"], inp2: Field[..., "float32"]):
         kind=common_types.ScalarKind.FLOAT32, shape=None
     )
     assert "nonlocal_unused" not in parsed.annex.symtable
-
-
-def test_external_symbols():
```

Comment:
Since we won't support this functionality please remove not only the test, but also the functionality for this. Namely `visit_ImportFrom` and the `externals_defs` field in `func_to_foast.py`.